src/test/java/io/javarig/generation/ObjectGenerationTest.java

```diff
@@ -165,7 +165,7 @@ public void shouldGenerateObjectWithFieldsContainingSetterPrefix() {
                 .extracting(ClassWithFieldContainingSetterPrefix::getSettersList)
                 .asList()
                 .isNotNull()
-                .hasSizeBetween(DefaultConfigValues.DEFAULT_MIN_SIZE_INCLUSIVE, DefaultConfigValues.DEFAULT_MAX_SIZE_EXCLUSIVE)// could be better if we had access to collectionGenerator defaultMin defaultMax size as public static fields
```

Comment:
we can remove the comment since we do have access to the defaul value as public static